app/components/Poll/index.tsx

```diff
@@ -183,6 +183,7 @@ class Poll extends Component<Props, State> {
                             {ratio < 18 && (
                               <span
                                 style={{
+                                  padding: '5px',
                                   marginLeft: '2px',
```

Comment:
Isn't this superfluous with the above change?

Reply:
It's not the same thing though? This was updated to make the alignment better, cause I thought it was affected by the change in `line-height`, but turns out that it was bad to begin with. Doesn't hurt to improve though.